test/prefer-string-starts-ends-with.mjs

```diff
@@ -70,29 +73,109 @@ test({
 			return {
 				code: `${re}.test(bar)`,
 				output: `bar.${method}('${string}')`,
-				errors: [{messageId}]
+				errors: [{
+					messageId,
+					suggestions: [
+						{
+							messageId: SUGGEST_STRING_CAST,
+							output: `String(bar).${method}('${string}')`
+						},
+						{
+							messageId: SUGGEST_OPTIONAL_CHAINING,
+							output: `bar?.${method}('${string}')`
+						},
+						{
+							messageId: SUGGEST_NULLISH_COALESCING,
+							output: `(bar ?? '').${method}('${string}')`
+						}
+					]
+				}]
 			};
 		}),
 		// Parenthesized
 		{
 			code: '/^b/.test(("a"))',
 			output: '("a").startsWith((\'b\'))',
-			errors: [{messageId: MESSAGE_STARTS_WITH}]
+			errors: [{
+				messageId: MESSAGE_STARTS_WITH,
+				suggestions: [
+					{
+						messageId: SUGGEST_STRING_CAST,
+						output: 'String("a").startsWith((\'b\'))'
+					},
+					{
+						messageId: SUGGEST_OPTIONAL_CHAINING,
+						output: '("a")?.startsWith((\'b\'))'
+					},
+					{
+						messageId: SUGGEST_NULLISH_COALESCING,
+						output: '(("a") ?? \'\').startsWith((\'b\'))'
+					}
+				]
+			}]
 		},
 		{
 			code: '(/^b/).test(("a"))',
 			output: '("a").startsWith((\'b\'))',
-			errors: [{messageId: MESSAGE_STARTS_WITH}]
+			errors: [{
+				messageId: MESSAGE_STARTS_WITH,
+				suggestions: [
+					{
+						messageId: SUGGEST_STRING_CAST,
+						output: '(String("a")).startsWith((\'b\'))' // TODO: remove extra parens around String()
```

Comment:
TODO: remove extra parens around String()

Reply:
I don't think we should, just keep it.

Reply:
Okay.